### PR TITLE
Added few documentation lines for the unsafeAddr operator.

### PR DESCRIPTION
**Pull request by krux02**
Originally this procedure just said "oh it is so unsafe to use this" without actually mentioning what needs to be taken care of when using it. I put some information the programmer needs to take care of.

### CODE REVIEW

**Comment by dom96** [comment]
Could you rephrase this, and maybe add an example? I read it a couple of times and I still don't understand it.

**Comment by krux02** [comment]
I read it again, just now, and it is exactly what I wrote. It basically says, read the documentation of whatever you are wrapping, before you use it. So unfortunately I can't give an example of, reading the documentation. I wanted to state, that one is in fact allowed to use this procedure, when the user makes sure that nothing will write to the content behind the pointer.
